sidekick_core/lib/sidekick_core.dart

```diff
@@ -34,13 +35,23 @@ export 'package:sidekick_core/src/sidekick_package.dart';
 ///
 /// Set [name] to the name of your CLI entrypoint
 ///
-/// [mainProjectPath], when set, links to the main package. For a flutter apps
-/// it is the package that actually builds the flutter app.
-/// Set [mainProjectPath] relative to the git repository root
+/// [mainProjectPath] should be set when you have a package that you
+/// consider the main package of the whole repository.
+/// When your repository contains only one Flutter package in root set
+/// `mainProjectPath = '.'`.
+/// In a multi package repository you might use the same when the main package
+/// is in root, or `mainProjectPath = 'packages/my_app'` when it is in a subfolder.
+///
+/// Set [flutterSdkPath] when you bind a flutter sdk to this project. This SDK
+/// enables the [flutter] and [dart] commands.
+/// [dartSdkPath] is inherited from [flutterSdkPath]. Set it only for pure dart
```

Comment:
We could add a check that the user doesn't set both `dartSdkPath` and `flutterSdkPath` (`assert(dartSdkPath == null || flutterSdkPath == null)`

Reply:
Technically that shouldn't be a problem